postgres/pgdump.go

```diff
@@ -259,84 +256,84 @@ func processAlterTableStmt(conv *internal.Conv, n nodes.AlterTableStmt) {
 		// track tables created by "CREATE TABLE", this lookup can fail.
 		// For debugging purposes we log the lookup failure if we're
 		// in verbose mode, but otherwise  we just skip these statements.
-		conv.SkipStatement(prNodes([]nodes.Node{n}))
-		internal.VerbosePrintf("Processing %v statement: table %s not found", reflect.TypeOf(n), table)
+		conv.SkipStatement(printNodeType(n))
+		internal.VerbosePrintf("Processing %v statement: table %s not found", printNodeType(n), table)
 	}
 }
 
-func processCreateStmt(conv *internal.Conv, n nodes.CreateStmt) {
+func processCreateStmt(conv *internal.Conv, n *pg_query.CreateStmt) {
 	var colNames []string
 	colDef := make(map[string]schema.Column)
 	if n.Relation == nil {
 		logStmtError(conv, n, fmt.Errorf("relation is nil"))
 		return
 	}
-	table, err := getTableName(conv, *n.Relation)
+	table, err := getTableName(conv, n.Relation)
 	if err != nil {
 		logStmtError(conv, n, fmt.Errorf("can't get table name: %w", err))
 		return
 	}
-	if len(n.InhRelations.Items) > 0 {
+	if len(n.InhRelations) > 0 {
 		// Skip inherited tables.
-		conv.SkipStatement(prNodes([]nodes.Node{n}))
+		conv.SkipStatement(printNodeType(n))
 		conv.Unexpected(fmt.Sprintf("Found inherited table %s -- we do not currently handle inherited tables", table))
-		internal.VerbosePrintf("Processing %v statement: table %s is inherited table", reflect.TypeOf(n), table)
+		internal.VerbosePrintf("Processing %v statement: table %s is inherited table", printNodeType(n), table)
 		return
 	}
 	var constraints []constraint
-	for _, te := range n.TableElts.Items {
-		switch i := te.(type) {
-		case nodes.ColumnDef:
-			name, col, cdConstraints, err := processColumn(conv, i, table)
+	for _, te := range n.TableElts {
+		switch te.GetNode().(type) {
+		case *pg_query.Node_ColumnDef:
+			name, col, cdConstraints, err := processColumn(conv, te.GetColumnDef(), table)
 			if err != nil {
 				logStmtError(conv, n, err)
 				return
 			}
 			colNames = append(colNames, name)
 			colDef[name] = col
 			constraints = append(constraints, cdConstraints...)
-		case nodes.Constraint:
+		case *pg_query.Node_Constraint:
 			// Note: there should be at most one Constraint node in
 			// n.TableElts.Items. We don't check this. We just keep
```

Comment:
Should this be just "n.TableElts" now?
i.e. drop the .Items?

Reply:
Done @https://github.com/cloudspannerecosystem/harbourbridge/pull/172/commits/3044120ca85a3b4386f12ca5a583fe4436cd0c54.